frontend/src/pages/lmEval/utils.ts

```diff
@@ -1,4 +1,13 @@
-import { LmEvalFormData } from './types';
+import { LmEvalFormData, LmModelArgument } from './types';
 
 export const isFilledLmEvalFormData = (data: LmEvalFormData): boolean =>
   data.tasks.length > 0 && !!data.model.name && !!data.model.url && !!data.model.tokenizer;
+
+export const convertModelArgs = (modelArgs: LmModelArgument): { name: string; value: string }[] => [
+  { name: 'model', value: modelArgs.name },
+  { name: 'base_url', value: modelArgs.url },
+  { name: 'num_concurrent', value: '1' },
+  { name: 'max_retries', value: '3' },
```

Comment:
Are we hardcoding these values?

Reply:
![Screenshot 2025-06-03 at 3 26 02 PM](https://github.com/user-attachments/assets/212ca7a1-2f68-4b34-bcd6-fd996ccdcc6a)
Its mentioned here defaults
for MVP we are keeping it defaults